server/cloudformation/apps.go

```diff
@@ -0,0 +1,116 @@
+package cloudformation
+
+import (
+	"github.com/jinzhu/gorm"
+	"github.com/remind101/empire"
+	"golang.org/x/net/context"
+)
+
+type AppsProvisioner struct {
+	empire *empire.Empire
+}
+
+func (p *AppsProvisioner) Provision(req Request) (id string, data interface{}, err error) {
+	ctx := context.Background()
+	user := &empire.User{Name: "cloudformation"}
```

Comment:
Cool thing is you'll still get even notifications when CloudFormation sets environment variables.